store/postgres/src/store.rs

```diff
@@ -823,6 +828,66 @@ impl SubgraphDeploymentStore for Store {
                 .map_err(|()| format_err!("deployment event notification listener failed")),
         )
     }
+
+    fn is_deployed(&self, id: &SubgraphId) -> Result<bool, Error> {
+        use db_schema::subgraph_deployments::dsl::*;
+
+        // The subgraph of subgraphs is always deployed.
+        if id == &SubgraphId::new(SUBGRAPHS_ID).unwrap() {
+            return Ok(true);
+        }
+
+        let deployment_count = subgraph_deployments
+            .filter(subgraph_id.eq(id.to_string()))
+            .execute(&*self.conn.lock().unwrap())?;
+        Ok(match deployment_count {
+            0 => false,
+            1 => true,
+            _ => unreachable!(), // The `subgraph_id` column is `UNIQUE`
+        })
+    }
+
+    fn schema_of(&self, subgraph_id: SubgraphId) -> Result<Schema, Error> {
```

Comment:
I'd prefer `subgraph_schema` here, just because you don't have to wonder what the schema is `of` when reading code that uses this function.

Reply:
Sounds good.